src/multi-selector.js

```diff
@@ -248,3 +246,13 @@ angular.module('cp-multi-selector')
 			}
 		}
 	}]);
+
+	function safeApply(scope, fn) {
```

Comment:
Kinda a hack to test if a digest is already in progress. If it is, don't `$apply`